views/halloffame.html

```diff
@@ -25,13 +26,51 @@
 				<h2>Golden Conversations</h2>
 				<div id="conversation-entries">
 					<div id="golden-conversations">
+						<div class="convo">
+							<h3><a href="#">An Amazing Conversation</a></h3>
+							<span class="name">Tony Jing</span><span class="name">Nikil Selvam</span><span class="name">Erik L</span>
+						</div>
+						<div class="convo">
+							<h3><a href="#">A Truly Golden Conversation</a></h3>
+							<span class="name">Tony Jing</span><span class="name">Nikil Selvam</span><span class="name">Erik L</span>
+						</div>
+					</div>
+					<div id="gold-plated-conversations">
+						<div class="convo">
+							<h3><a href="#">How much wood would a woodchuck chuck?</a></h3>
+							<span class="name">Tony Jing</span><span class="name">Nikil Selvam</span><span class="name">Erik L</span><span class="name">Woody Woodchuck</span>
+						</div>
+						<div class="convo">
+							<h3><a href="#">The President's Choice</a></h3>
+							<span class="name">Theodore Roosevelt</span><span class="name">Abraham Lincoln</span><span class="name">Thomas Jefferson</span>
+							<span class="name">Franklin D. Roosevelt</span>
+						</div>
 					</div>
 					<div id="saved-conversations">
+						<div class="convo">
+							<h3><a href="#">Your best moments of 2014</a></h3>
+							<span class="name">Lebron James</span><span class="name">Futebol do Brasil</span><span class="name">L. Messi</span>
+							<span class="name">Mihaly Csikszentmihalyi</span>
+						</div>
+						<div class="convo">
+							<h3><a href="#">What books have influenced you?</a></h3>
+							<span class="name">Charles Xavier</span><span class="name">Leonardo Da Vinci</span><span class="name">Flying Dutchman</span>
+							<span class="name">Ang Lee</span>
+						</div>
 					</div>
 				</div>
 
 			</div>
 		</section>
+		<div class="right-container">
+			<section class="hof-helper right-side">
+				<h2>What are <br><span>Golden<br> Conversations?</span></h2>
+				<p>Conversations that <span class="hof-helper-emphasis">all</span> participants have added to the Hall of Fame are <span class="gold-label">golden</span>. Conversations with <span class="hof-helper-emphasis">one or more</span> participants adding to the HOF are <span class="goldplated-label">gold plated</span>. For conversations only you’ve added to the HOF, there is no gold. So Ask the other participants to add them so they too can become golden!</p>
```

Comment:
I think both of you guys have some great feedback here. I would combine your thoughts and make changes accordingly. For sure the future tense might be better, especially when you first come in to the HOF and sees nothing but the helper text.